packages/remix/src/ssr/rootAuthLoader.ts

```diff
@@ -33,7 +33,10 @@ export const rootAuthLoader: RootAuthLoader = async (
   const { authData, showInterstitial } = await getAuthData(args.request, opts);
 
   if (showInterstitial) {
-    throw json(wrapClerkState({ __clerk_ssr_interstitial: showInterstitial, __frontendApi: frontendApi }));
+    throw json(
+      wrapClerkState({ __clerk_ssr_interstitial: showInterstitial, __frontendApi: frontendApi }), 
+      {headers: {"Cache-Control": "no-cache"}}
```

Comment:
As discussed and in order to be consistent across similar implementations (Nextjs), instead of adding a `Cache-Control` header explicitly, shall we change the status of this response to `401`?

```tsx
    throw json(wrapClerkState({ __clerk_ssr_interstitial: showInterstitial, __frontendApi: frontendApi }), {
      status: 401,
    });
```

The same change should be made to: https://github.com/clerkinc/javascript/blob/acb818ce77b036690845763a45471ebf5229de49/packages/remix/src/ssr/getAuth.ts#L18

I'd be happy to do the change myself, but I don't want to hijack your PR :)